hybrid/ddc_invalid.c

```diff
@@ -0,0 +1,19 @@
+/***
+ * This program causes an "In-address space security Exception"
+ * by clearing the tag of the capability in the DDC
```

Comment:
Where is the exception caused? It might be necessary to split line 17 into separate parts so that you can be clear about which causes the exception.

Reply:
This should have been fixed by the assertions in 7548354.